docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -522,48 +538,49 @@ This is done by ensuring tokens can be trusted.
 [[token-verification-introspection]]
 ==== Token verification and introspection
 
-The verification process of OIDC authorization code flow tokens follows the Bearer token authentication token verification and introspection logic. For more information, see the xref:security-oidc-bearer-token-authentication.adoc#token-verification-introspection[Token Verification And Introspection] section of the "Quarkus OpenID Connect (OIDC) Bearer token authentication" guide.
+The verification process of OIDC authorization code flow tokens follows the Bearer token authentication token verification and introspection logic.
+For more information, see the xref:security-oidc-bearer-token-authentication.adoc#token-verification-introspection[Token verification and introspection] section of the "Quarkus OpenID Connect (OIDC) Bearer token authentication" guide.
 
 [NOTE]
---
-With Quarkus `web-app` applications, only the `IdToken` is verified by default because the access token is not used to access the current Quarkus `web-app` endpoint and is intended to be propagated to the services expecting this access token.
-If you expect the access token to contain the roles required to access the current Quarkus endpoint (`quarkus.oidc.roles.source=accesstoken`) then it will also be verified.
---
+===
```

Comment:
Sometimes I'm seeing `====` with the Note, is `===` correct ?

Reply:
Good catch. The correct formatting is "====" and not "===" in order for the Note to render correctly (and not display "===".